.github/workflows/tail-call.yml

```
@@ -79,19 +79,17 @@ jobs:
         with:
           python-version: '3.11'
 
-      - name: Native Windows (debug)
+      - name: Native Windows MSVC (release)
         if: runner.os == 'Windows' && matrix.architecture != 'ARM64'
         shell: cmd
```

A forthcoming update of the zizmor CI security scanner (in pre-commit) will complain about `shell: cmd`:

> The [obfuscation](https://docs.zizmor.sh/audits/#obfuscation) audit now detects usages of shell: cmd and similar, as the Windows CMD shell lacks a formal grammar and limits analysis of run: blocks in other audits (https://github.com/zizmorcore/zizmor/issues/1361)

https://github.com/zizmorcore/zizmor/releases/tag/v1.17.0

Shall we already convert this to `pwsh`?

```suggestion
        shell: pwsh
```

---

Let's try. I'll change the other one below, too.

---

i don't think we can. The problem is powershell confuses the build script when we pass in arguments to MSBuild :(

---

Actually I'm not so sure now, there's probably a way to get around this?

---

Me neither, but CI is red now. Reverting to cmd for now ...

Can be done in a follow up PR?

---

Yeah I'd prefer this be done in a follow-up PR.

---

Followup in PR https://github.com/python/cpython/pull/143095.